protozfits/digicam.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from astropy.utils.decorators import lazyproperty
+
+from .simple import File
+from .patch_ids import PATCH_ID_INPUT, PATCH_ID_OUTPUT
+
+
+def event_source(path, run_id=0):
+    for event in File(path).Events:
+        yield Event(event, run_id)
+
+
+class Event:
+    def __init__(self, event, run_id):
+        self.run_id = run_id
+        self._event = event
```

Comment:
we have three variables which are identicall here:
```python
event
self._event = event
_e = self._event
```
Think we should just keep : `self._event`

Reply:
`_e` and `_w` are just used to make lines shorter, but they may be confusing.

Short variable names are bad, because the reader has now idea what is going on...
**but** if the bad variable name is used only in a small scope, the reader has a chance.

In this case I thought, the short variable name actually improves readability, maybe you could make a different proposal and push it directly here?

Reply:
okay .. worked ... one can get rid of _e and _w entirely.. thanks